recipes/p-sea/build.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/sh
+
+#strictly use anaconda build environment
+CC=${PREFIX}/bin/gcc
+CXX=${PREFIX}/bin/g++
+
+./configure --prefix=$PREFIX
```

Comment:
The error message indicates that there is no `configure` script.
https://travis-ci.org/bioconda/bioconda-recipes/jobs/272909231#L827